### PR TITLE
Added support for simple and automated endpoint for RP and API

### PR DESCRIPTION
**Pull request by dmennis**
Added Xcode build configuration file (Constants.xcconfig) to store the Relying Party ID and API endpoint URL. This allows the developer to only update one file and automatically picks up the associated domain from the RP ID environment variable.

### CODE REVIEW

**Comment by dmennis** [comment]
Let me know if it's better to change the defaults below back to the generic 'your_url_endpoint_cloudflare.com'
// API Endpoint URL
API_BASE_URL = api.pawskey.co/v1

// Relying Party Endpoint Domain. This needs to match your RP Id
RP_ID = pawskey.co

File: examples/clients/mobile/iOS/PawsKey/Constants.xcconfig [hunk 1]
```diff
@@ -0,0 +1,13 @@
+//
+//  Constants.xcconfig
+//  PawsKey
+//
+//  Created by Dennis Hills on 6/5/23.
+//  Copyright © 2023 Yubico. All rights reserved.
+//
+
+// API Endpoint URL
+API_BASE_URL = api.pawskey.co/v1
```
Comment (joostd) [inline examples/clients/mobile/iOS/PawsKey/Constants.xcconfig:10]: Perhaps better make this a URL i.e.

API_BASE_URL = https://api.pawskey.co/v1

File: examples/clients/mobile/iOS/PawsKey/Shared/Constants.swift [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            return value
+        default:
+            throw Error.invalidValue
+        }
+    }
+}
+
+enum API {
+    static var baseURL: String {
+        return try! "https://" + Constants.value(for: "API_BASE_URL")
```
Comment (joostd) [inline examples/clients/mobile/iOS/PawsKey/Shared/Constants.swift:34]: See previous comment on API_BASE_URL

**Review by dmennis — COMMENTED** [review]
Reviewed with Joost. Approved.

**Comment by dmennis** [comment]
Merge.

**Pull request merged**